packages/ckb/CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # @rgbpp-sdk/ckb
 
+## 0.7.0
+
+### Minor Changes
+
+- [#291](https://github.com/utxostack/rgbpp-sdk/pull/291): Support RGB++ compatible xUDT assets([@duanyytop](https://github.com/duanyytop))
+
+- [#293](https://github.com/utxostack/rgbpp-sdk/pull/293): Add offline btc data source & ckb collector ([@fghdotio](https://github.com/fghdotio))
+
+- [#294](https://github.com/utxostack/rgbpp-sdk/pull/294): Add UTXOAirdropBadge Testnet deployment information ([@duanyytop](https://github.com/duanyytop))
+
+- [#297](https://github.com/utxostack/rgbpp-sdk/pull/297): Add USDI to compatible xUDT list ([@duanyytop](https://github.com/duanyytop))
+
+- [#302](https://github.com/utxostack/rgbpp-sdk/pull/302): Encapsulate transaction fee supplementation methods ([@fghdotio](https://github.com/fghdotio))
+
+- [#303](https://github.com/utxostack/rgbpp-sdk/pull/303): Add offline mode support for compatible xUDT type scripts ([@fghdotio](https://github.com/fghdotio))
+
+- [#304](https://github.com/utxostack/rgbpp-sdk/pull/304): Remove duplicate cell deps ([@duanyytop](https://github.com/duanyytop))
```

Comment:
PR 304 doesn't include many changes; 
it seems the changelog is messed up, with unrelated changelogs packed into this one.

Reply:
Updated in https://github.com/utxostack/rgbpp-sdk/pull/310/commits/a2a311f30bf387fd3bf3aa159da43c45bbdcce66